docs/run-preinstalled-wda.md

```diff
@@ -0,0 +1,81 @@
+---
+title: Run Preinstalled WebDriverAgentRunner
+---
+
+XCUITest driver can launch preinstalled WebDriverAgent directly against a real device.
+It lets you start a XCUITest driver session without the `xcodebuild` command execution to improve the session startup performance.
+
+## For Real Devices
+
+### Capabilities
+
+- Required
+  - `appium:usePreinstalledWDA` to start a WebDriverAgent Runner-process without xcodebuild
+- Optional
+  - `appium:updatedWDABundleId` to customize the WebDriverAgent bundle id XCUITest driver launches
+
+### Example steps with Xcode
+
+Open WebDriverAgent in Xcode
```

Comment:
Please merge these steps with the below one into a single numbered list.
We could also mention that the `appium driver run xcuitest open-wda` command could be used to quickly open WDA in Xcode

Reply:
oops